tests/data/user_expectations.json

```diff
@@ -167,6 +167,13 @@
     {"user": "You", "variation": "Variation-1"},
     {"user": "Zeba", "variation": "Variation-1"}
   ],
+  "AB_T_100_W_25_25_25_25": [
+    {"user": "Faizan", "variation": "Control"},
```

Comment:
the list of users is wrong.
We should use the same 26 users as used in other test cases.

These names were just for your reference

Reply:
cool.

Reply:
Done.